for-upstream/default.nix-darwin.nix

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
What's the benefit of having a copy here?

Reply:
We can let people use a specific version of the module (stable, master, feature branch for experimentation)

It's easier to maintain consistency when all files are in one place. Upstreaming is `cp` + `git diff`, which will also reveal any changes made by upstream, giving an opportunity to incorporate those changes into other deployment targets where relevant.